README.md

```diff
@@ -62,6 +62,22 @@ http://127.0.0.1:5000/apidocs/ by default.
 The API server will read its configuration on boot from the .env file located in
 `src/rhelocator/api/.env`.
 
+### How to build the API container on your local system
+1. Build the container from within the rhelocator directory:
+```
+buildah bud -f ./Containerfile --format docker --tls-verify=true -t rhelocator:latest .
```

Comment:
My PC tells me, `buildah` is not installed. :smile: What is the difference to `podman build`?
```suggestion
podman build -f Containerfile -t rhelocator:latest .
```

Reply:
Yep both tools are compatible ;) . I think summing up the entire buildah system is a bit too much for a comment but here is the 101: Buildah and podman only differ slightly. Builder only cares about creating OCI (Open Container Initiative) images while podman takes care of the entire lifecycle. Buildah comes with a couple of advantages, like being able to build images that you can run in different container environments without necessarily having something like a docker daemon running on your system. There is plenty more but I think the docs tell it best: https://www.redhat.com/en/topics/containers/what-is-buildah

For your local builds it probably doesn't matter though which you use for building! 
I think I'll add the podman command as well

Reply:
Unfortunately I still don't understand why we can't use podman to build the image?

We use buildah to use the docker format instead of the OCI format[1], right? From what I read, the docker format is the OCI format. 

> Docker's original image format has become the OCI Image Specification, and various open-source build tools support it, including:
>    - BuildKit, an optimized rewrite of Docker's build engine;
>    - Podman, an alternative implementation of Docker's command-line tool;
>    - Buildah, a command-line alternative to writing Dockerfiles;
> Given an OCI image, any container runtime that implements the OCI Runtime Specification can unbundle the image and run its contents in an isolated environment. Docker donated its runtime, runc, to the OCI to serve as the first implementation of the standard.

1: https://www.padok.fr/en/blog/container-docker-oci

Reply:
Oh, you can use podman just fine (or Docker for that matter) and yes you are completely correct OCI is the default docker format. It's just an example of how you can build the container on your local system for the readme. As Buildah is part of my workflow I used it as an example, that's it ;) . You can use whatever you are comfortable with, but I will delete the buildah line form the readme if that helps!

Reply:
@miyunari Oh noooooooo I just realized what the confusion was all about. I did not see the `--format` flag in my command (I'd like the blame it on autocomplete but I'm sure it was more a copy-and-paste thing :( ).
Yes you are absolutely right about it making no sense to have that in the docs :) . We want to use OCI and not specify the docker format explicitly (but that's up to who ever wants to build that container). I'll remove that flag ASAP.

About the buildah / podman build thing, I still don't see a harm in having both examples in there but I'm not sad if we remove it. What do you think?

Reply:
@miyunari I've merged your suggestion, thanks for pointing this out! Guess I need to work on my reading skills.